Channels/Integration/02_ManageVariantSets.md

```diff
@@ -0,0 +1,184 @@
+[!!User interface Variant sets](../UserInterface/to_be_completed)
+[!!Manage the variant sets in PIM](../../PIM/Integration/07_ManageVariantSets.md)
+
+
+# Manage a variant set
+
+The variant sets in *Omni-Channel* are based on the same data model as in the Actindo *PIM* module. However, since the *Omni-Channel* and the Actindo *PIM* module are independent modules and *Omni-Channel* can also be used without the Actindo *PIM* module, the variant sets for products can also be managed within the *Omni-Channel* module.    
```

Comment:
```suggestion
The variant sets in *Omni-Channel* are based on the same data model as in the Actindo *PIM* module. However, since the *Omni-Channel* and the Actindo *PIM* modules are independent of each other and can be used separately, the variant sets for products can also be managed within the *Omni-Channel* module.    
```